### ISSUE TITLE
Graylog should stop processing on ClusterBlockException[blocked by: [FORBIDDEN/8/index write (api)];]

### ISSUE THREAD

**Issue by pdepaepe** [Graylog2/graylog2-server#1609]
Hello,

Trying to get the ES hard drive almost full, i got tons of theses dead letter:

```
[48]: index [graylog2_1818], type [message], id [a9fe76ee-9cc3-11e5-a827-56847afe9799], message [ClusterBlockException[blocked by: [FORBIDDEN/8/index write (api)];]]
```

Which mean ES have no more size to handle documents. Problem here, graylog continue to stream to it and so loose messages in dead letters.

I think, Graylog should block "Node processing" with a red alert on "Overview" page when this occur as without human intervention it can't be solved.

Thanks

**Comment by bernd** [comment]
This will be fixed with the upcoming Graylog 2.3. We are now using the HTTP transport to index into Elasticsearch and a ClusterBlockException will now trigger a retry instead of discarding the messages.

**Comment by bernd** [comment]
We had to change the retry behavior shortly before 2.3 so this can still be an issue. Please follow https://github.com/Graylog2/graylog2-server/issues/2211 for updates on this.
